e2e/framework/helper.go

```diff
@@ -416,7 +416,7 @@ func NewTestServers(t *testing.T, num int, conf func(*TestServerConfig)) []*Test
 	})
 
 	// It is safe to use a dummy MultiAddr here, since this init method
-	// is called for Dev / Dummy consensus modes, and IBFT servers are initialized with NewIBFTServersManager.
+	// is called for the Dev consensus mode, and IBFT servers are initialized with NewIBFTServersManager.
```

Comment:
Remove comment about the Dev consensus, it doesn't exist as of this PR

Reply:
This PR only removes the `dev` **command**, the **dev consensus** is still present and used throughout the codebase

Reply:
My bad